jbmc/src/java_bytecode/java_types.h

```diff
@@ -233,22 +285,16 @@ class java_class_typet:public class_typet
     }
   };
 
-  using componentst = std::vector<componentt>;
+  using methodst = std::vector<methodt>;
 
-  const componentst &components() const
+  const methodst &methods() const
```

Comment:
Are there any users for this? It'd be good to include at least some in this PR so doesn't get deleted Perhaps a unit test for checking that a java class when loaded has the methods set?

Reply:
Coming!